_test/test_geometry/test_4D_ort.m

```diff
@@ -0,0 +1,96 @@
+classdef test_4D_ort < TestCase
```

Comment:
I've added description, which hopefully better explains what the test is doing. It may be my "russian" slang (nor actually Russian, its rather math-russian), where ort always means orthogonality....